lte/gateway/python/magma/subscriberdb/client.py

```diff
@@ -119,10 +131,50 @@ async def _check_subscribers_in_sync(self) -> bool:
             return False
         return res.in_sync
 
-    async def _get_all_subscribers(self) -> (SubscriberData, Digest):
+    async def _sync_subscribers(self) -> bool:
+        """
+        Sync local subscriber data and digests with the cloud if didn't receive
+        resync signal.
+
+        Returns:
+            boolean value for whether a resync with cloud is needed
+        """
+        subscriberdb_cloud_client = self._grpc_client_manager.get_client()
+        req = SyncSubscribersRequest(
+            per_sub_digests=self._store.get_current_per_sub_digests(),
+        )
+        try:
+            res = await grpc_async_wrapper(
+                subscriberdb_cloud_client.SyncSubscribers.future(
+                    req,
+                    self.SUBSCRIBERDB_REQUEST_TIMEOUT,
+                ),
+                self._loop,
+            )
+        except grpc.RpcError as err:
+            logging.error(
+                "Sync subscribers request error! [%s] %s", err.code(),
+                err.details(),
+            )
+            return True
+
+        if not res.resync:
+            self._update_flat_digest(res.flat_digest)
+            self._update_per_sub_digests(res.per_sub_digests)
+
+            for sid in res.to_renew:
+                subscriber_data = res.to_renew[sid]
+                self._store.upsert_subscriber(subscriber_data)
+            for sid in res.deleted:
+                self._store.delete_subscriber(sid)
```

Comment:
Edit: I see that this is how we were doing it before. Let's keep it as-is then, and we can defer the bulk-edit functionality to a future todo. Can you add a `TODO(hcgatewood)`?

This could be up to 500 subscribers -- instead of doing one at a time, can we bulk-upsert and/or bulk-delete?